src/commands/autocomplete/index.ts

```diff
@@ -35,15 +39,33 @@ export default class Index extends AutocompleteBase {
     if (!flags['refresh-cache']) {
       const bin = this.config.bin
       const tabStr = shell === 'bash' ? '<TAB><TAB>' : '<TAB>'
-      const note = shell === 'zsh' ? `After sourcing, you can run \`${chalk.cyan('$ compaudit -D')}\` to ensure no permissions conflicts are present` : 'If your terminal starts as a login shell you may need to print the init script into ~/.bash_profile or ~/.profile.'
+
+      const instructions = shell === 'powershell' ?
+        `New-Item -Type Directory -Path (Split-Path -Parent $PROFILE) -ErrorAction SilentlyContinue
+Add-Content -Path $PROFILE -Value (Invoke-Expression -Command "${bin} autocomplete${this.config.topicSeparator}script ${shell}"); .$PROFILE` :
+        `$ printf "eval $(${bin} autocomplete${this.config.topicSeparator}script ${shell})" >> ~/.${shell}rc; source ~/.${shell}rc`
+
+      let note = ''
+
+      switch (shell) {
+      case 'zsh':
+        note = `After sourcing, you can run \`${chalk.cyan('$ compaudit -D')}\` to ensure no permissions conflicts are present`
+        break
+      case 'bash':
+        note = 'If your terminal starts as a login shell you may need to print the init script into ~/.bash_profile or ~/.profile.'
+        break
+      case 'powershell':
+        note = `Use the \`MenuComplete\` mode to get matching completions printed below the command line:\n${chalk.cyan('Set-PSReadlineKeyHandler -Key Tab -Function MenuComplete')}`
```

Comment:
pwsh has 3 completion modes (`Complete`, `TabCompleteNext` and `MenuComplete`).
I found `MenuComplete` to be the most useful (interactive menu, works like zsh) so added it here as a suggestion.